multicorn_das/__init__.py

```diff
@@ -252,7 +252,7 @@ def execute(self, quals, columns, sortkeys=None, limit=None, planid=None):
             das_id=self.das_id,
             table_id=self.table_id,
             query=query,
-            plan_id=str(planid),
+            plan_id=planid,
```

Comment:
If you leave it as it was, it is harmless, right? Would work both for unique number and string. If so, I'd close this PR.

Reply:
Indeed it's working the same:
```
plan_id: "0033-6"
```